docs/docs/configuration/databases.mdx

```diff
@@ -1524,6 +1525,78 @@ Other parameters:
 - Load Balancer - Backup Host
 
 
+
+#### YDB
+
+The recommended connector library for [YDB](https://ydb.tech/) is
+[ydb-sqlalchemy](https://pypi.org/project/ydb-sqlalchemy/).
+
+##### Connection String
+
+The connection string for YDB looks like this:
+
+```
+ydb://{host}:{port}/{database_name}
+```
+
+##### Protocol
+You can specify `protocol` in the `Secure Extra` field at `Advanced / Security`:
+
+```
+{
+    "protocol": "grpcs"
+}
+```
+
+Default is `grpc`.
+
+
+##### Authenticaions
```

Comment:
Typo here

Reply:
Actually it ought to say "authentication methods" or something similar really since authentication itself can't be pluralised.

Reply:
fixed